float32 stepHeight
float32 stepLength
float32 stepTime

